### PR TITLE
Payment methods page

### PR DESCRIPTION
**Pull request by Vadim-Kostetskyi**

### CODE REVIEW

File: src/modules/checkout/components/Payment/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+import Paypal from 'assets/images/payment/paypal.png';
+import ApplePay from 'assets/images/payment/apple-pay.png';
+import styles from './index.module.scss';
+
+const Payment = () => {
+  const { t } = useTranslation();
+  const paymentChoice = (e: React.MouseEvent<HTMLButtonElement>) => {
+    //TODO add the function of choosing a payment method
+    console.log(e.currentTarget.name);
+  };
```
Comment (denred) [inline src/modules/checkout/components/Payment/index.tsx:17]: I thought it should open a modal for entering payment information.

**Review by denred — APPROVED** [review]

File: src/modules/checkout/components/Payment/index.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import PaymentMethodButton from 'modules/checkout/components/PaymentMethodButton';
+
+const Payment = () => {
+  const { t } = useTranslation();
+  const paymentChoice = (e: React.MouseEvent<HTMLButtonElement>) => {
+    //TODO add the function of choosing a payment method
+    console.log(e.currentTarget.name);
+  };
+
+  const paymentMethodButtons = [
```
[the remark this answers is not in the document]
Comment (Vadim-Kostetskyi) [reply src/modules/checkout/components/Payment/index.tsx:20]: Yes, it is in the styles

[the remark this answers is not in the document]
Comment (Vadim-Kostetskyi) [reply src/modules/checkout/components/Payment/index.tsx:20]: Yes, it is in the layout

[the remark this answers is not in the document]
Comment (Vadim-Kostetskyi) [reply src/modules/checkout/components/Payment/index.tsx:20]: there is already Discover:     {
      title: t('order.creditCard'),
      name: 'Visa',
      images: [
        { link: Visa, alt: 'Visa image' },
        { link: Mastercard, alt: 'Mastercard image' },
        { link: AmericanExpress, alt: 'AmericanExpress image' },
        { link: Discover, alt: 'Discover image' },
      ],
    },

**Pull request merged**